resourcemonitor/resourcemonitor.go

```diff
@@ -175,27 +193,32 @@ func New(
 	if monitor.config.RAM != nil {
 		monitor.alertProcessors.PushBack(createAlertProcessor(
 			"System RAM",
-			&monitor.globalMonitoringData.RAM,
+			&monitor.nodeMonitoringData.RAM,
 			func(time time.Time, value uint64) {
 				monitor.alertSender.SendAlert(prepareSystemAlertItem("ram", time, value))
 			},
 			*monitor.config.RAM))
 	}
 
 	if monitor.config.UsedDisk != nil {
-		monitor.alertProcessors.PushBack(createAlertProcessor(
-			"System Disk",
-			&monitor.globalMonitoringData.UsedDisk,
-			func(time time.Time, value uint64) {
-				monitor.alertSender.SendAlert(prepareSystemAlertItem("disk", time, value))
-			},
-			*monitor.config.UsedDisk))
+		monitor.nodeMonitoringData.Disk = make([]cloudprotocol.PartitionUsage, len(config.Partitions))
```

Comment:
Wrong place to initialize Disk here. If disk alerts are not enabled, disk statistics will not be provided. But this is not expected behavior.

Reply:
Done